src/components/FeatureFlag.tsx

```diff
@@ -26,9 +52,18 @@ const FeatureFlag = (
 
     let isEnabled = defaultValue;
 
-    // set the flag value if found and check if inverted
     if (flag !== undefined) {
-        isEnabled = invert ? !flag.enabled : flag.enabled;
+
+        if (flag.enabled && flag.strategies) {
+            // check if strategies match
+            const flagStrategyResult = checkFlagStrategies(flag, flagsClient ? flagsClient.getUserId() : undefined);
```

Comment:
No need for a new variable here. 
`isEnabled = checkFlagStrategies(flag, flagsClient ? flagsClient.getUserId() : undefined);`